### PR TITLE
RD-4414 Add tokens endpoint handling

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

File: cloudify_cli/tests/commands/test_tokens.py [hunk 1]
```diff
@@ -1,21 +1,51 @@
...
 
-    def test_tokens_command(self):
-        self.use_manager()
-        self.invoke('cfy tokens get')
+    def test_old_command_error(self):
+        # Make sure we give a helpful error on the old style token retrieval
+        self.invoke('cfy tokens get', err_str_segment='`cfy tokens create`')
+
+    def _check_output(self, tokens, output, expect_users=False,
+                      expect_count=True):
```
[the remark this answers is not in the document]
Comment (geokala) [reply cloudify_cli/tests/commands/test_tokens.py:26]: They should already be there.

[the remark this answers is not in the document]
Comment (geokala) [reply cloudify_cli/tests/commands/test_tokens.py:26]: (and will be shortly)

File: cloudify_cli/tests/commands/test_tokens.py [hunk 2]
```diff
@@ -1,21 +1,98 @@
...
 
 class TokensTest(CliCommandTest):
 
     def setUp(self):
         super(TokensTest, self).setUp()
-        self.client.tokens.get = MagicMock()
+        self.client.tokens.get = Mock()
+        self.client.tokens.list = Mock()
+        self.client.tokens.delete = Mock()
+        self.client.tokens.create = Mock()
```
Comment (tehasdf) [inline cloudify_cli/tests/commands/test_tokens.py:25]:
```suggestion
        self.client.tokens = Mock()
```

Comment (geokala) [reply cloudify_cli/tests/commands/test_tokens.py]: Updated

**Review by tehasdf — APPROVED** [review]

**Pull request merged**